internal/orchestrationapi/orchestrationapi.go

```diff
@@ -156,7 +156,7 @@ func (orcheEngine *orcheImpl) RequestService(serviceInfo ReqeustService) Respons
 
 	log.Printf("[RequestService] getCandidate")
 	for index, candidate := range candidates {
-		log.Printf("[%d] Id       : %v", index, candidate.Id)
+		log.Printf("[%d] Id       : %v", index, candidate.ID)
```

Comment:
The variable ID comes from ExecutionCandidate structure from helper.go file. This file generated the following lint warning
internal/db/helper/helper.go:37:2: struct field Id should be ID
To remove this warning, the variable was changed to ID from Id. Hence the change in variable. Log message it can be reverted if required :)

Reply:
Please take a closer look at my suggestion.

Reply:
Done!